source/unified-test-format/unified-test-format.md

```diff
@@ -0,0 +1,3459 @@
+`ignoreExtraEvents`.
+
+### Target failPoint by read preference
+
+The [failPoint](#failpoint) operation currently uses a "primary" read preference. To date, no spec has needed behavior
+to configure a fail point on a non-primary node. If the need does arise, [failPoint](#failpoint) can be enhanced to
+support a `readPreference` argument.
+
+### IO operations for GridFS streams
+
+Original GridFS spec tests refer to "upload", "download", and "download_by_name" methods, which allow the tests to
+abstract stream IO and either upload a byte sequence or assert a downloaded byte sequence. These operations correspond
+to the [download](#download), [downloadByName](#downloadByName), [upload](#upload-and-uploadWithId), and
+[uploadWithId](#upload-and-uploadWithId) [Bucket Operations](#bucket-operations).
+
+In order to support methods such as `downloadToStream`, `openUploadStream`, and `openUploadStreamWithId`, test runners
+would need to represent streams as entities and support IO operations to directly read from and write to a stream
+entity. This may not be worth the added complexity if the existing operations provide adequate test coverage for GridFS
+implementations.
+
+### Support Client-side Encryption integration tests
+
+Supporting client-side encryption spec tests will require the following changes to the test format:
+
+- `json_schema` will need to be specified when creating a collection, via either the collection entity definition or
+  [initialData](#initialData).
+- `key_vault_data` can be expressed via [initialData](#initialData)
+- `autoEncryptOpts` will need to be specified when defining a client entity. Preparation of this field may require
+  reading AWS credentials from environment variables.
+
+The process for executing tests should not require significant changes, but test files will need to express a dependency
+on mongocryptd.
+
+### Incorporate referenced entity operations into the schema version
+
+The [Schema Version](#schema-version) is not impacted by changes to operations defined in other specs and referenced in
+`Entity Test Operations` (e.g. `find` for CRUD). The [operation.name](#operation_name) and
+[operation.arguments](#operation_arguments) fields are loosely defined in the JSON schema as string and object types,
+respectively.
+
+Ideally, all operations (and their arguments) would be enforced by the JSON schema *and* any changes to operations would
+affect the schema version accordingly. For example, a new `find` option would warrant a minor version bump both for the
+CRUD spec and this spec and its schema.
+
+As discussed in [Executing an Operation](#executing-an-operation), test runners MUST raise errors for unsupported
+operations and arguments. This is a concession until such time that better processes can be established for versioning
+other specs *and* collating spec changes developed in parallel or during the same release cycle.
+
+## Changelog
+
+- 2024-02-06: Migrated from reStructuredText to Markdown.
```

Comment:
I overlooked this in previous PRs but there's no way we can retain [Definition Lists](https://www.markdownguide.org/extended-syntax/#definition-lists) here, since it's non-standard syntax, correct?

Reply:
Correct, it doesn't render:

First Term
: This is the definition of the first term.

Second Term
: This is one definition of the second term.
: This is another definition of the second term.